apps/common/views/partials/address-details.html

```diff
@@ -0,0 +1,24 @@
+<div>
+  <fieldset id="address-group" class="govuk-checkboxes__conditional govuk-checkboxes__conditional--hidden">
+    <br>
+    <p>{{#t}}pages.contact-details.address.title{{/t}}</p>
+
+    {{#input-text}}contact-address-house-number{{/input-text}}
+
+    {{#input-text}}contact-address-street{{/input-text}}
+
+    {{#input-text}}contact-address-town{{/input-text}}
+
+    {{#input-text}}contact-address-county{{/input-text}}
+
+    {{#input-text}}contact-address-postcode{{/input-text}}
+
+    <div class="govuk-warning-text">
```

Comment:
do you need all those new lines in between?

Reply:
removed new lines